dataland-frontend/tests/e2e/specs/data-request/DataRequestProcess.ts

```diff
@@ -202,7 +202,8 @@ describe("As a user I expect a data request page where I can download an excel t
         });
       });
 
-      it(`Test if the checkbox state is transferred correctly to the request`, () => {
+      //This test needs to be skipped until the anonymous data request feature is reintroduced
```

Comment:
Die Coding Guidelines sagen, dass der wohl ganz raus muss, fürs erste. Das fällt wohl in die Kategorie Dead Code

Reply:
Okay ist raus